doc/README.rst

```diff
@@ -3,52 +3,60 @@
 The doc folder
 ==============
 
-The documentation for NEST is contained in this folder. We are currently working
-on improving the documentation and making updates. However, if you notice
-something out of place, you can `submit an issue <https://nest.github.io/nest-simulator/development_workflow#reporting-bugs-and-issues>`_
-in our GitHub repository.
+This folder contains the static parts of the documentation for NEST,
+as well as the infrastructure and tools for extracting documentation
+dynamically from the source code and building a unified HTML site
+using Sphinx.
 
-Build the docs locally
------------------------
+For each of the releases, the fully built documentation is available
+at https://nest-simulator.readthedocs.io/
+
+Build the user documentation locally
+------------------------------------
 
-You will need the NEST source code in your project::
+If you intend to work on the documentation, or if you want to obtain a
+local version of it for other reasons, you can build the documentation
+by simply running
 
+::
 
-    cd nest-simulator/doc/
+    make html
 
-Requirements for building the docs include::
+in the build directory of NEST (i.e. the directory where you ran
+``cmake``). To install the dependencies for building the
+documentation, you can either directly run
 
- python3
- setuptools
- pandoc
- sphinx
- recommonmark
- sphinx-rtd-theme
- nbsphinx
- breathe
- sphinx-gallery
- ipython
- mock
+::
 
+    pip install -r <nest_source_dir>/doc/requirements.txt
 
-You can now build the docs using this command (make sure you are in the doc folder)::
+or alternatively install the dependencies into a conda environment by
+running
 
-    make html
+::
 
+    conda env create -f <nest_source_dir>/doc/environment.yml -p conda
```

Comment:
While this conda command works, I have to activate the environment from the source directory or use the absolute path name. This isn't a big deal but all my other conda environments are stored along a different path and can be activated from anywhere via their ENVNAME. Is this a potential area of confusion for users?

Reply:
Hi @jessica-mitchell! Nice to see you around :-)

If you leave out the `-p conda` in the line above, it will create the environment *globally* with the name `nest-doc` (c.f. [here](https://github.com/nest/nest-simulator/pull/1843/files#diff-178af50da4e5ad380ede922c14d44bc72486a29805cac3a1f56dfa0ae7aef21aR1)). I just thought that co~rona~nda-skeptics like me might prefer to have it in an obvious local directory, so I can be sure it'd just go away whenever I delete the build directory.

Would it help to have both variants explained in the `README.md`?

Reply:
I agree with @jessica-mitchell here and would remove the `-p conda`, since it quite strongly breaks with conda logic and user experience. The default for conda is a user-local installation, i.e., the complete conda installation lives in the users home directory.  Global installations for all users of a system are rare for all I know. So installing without `-p ...` will install for the user only, and then automatically in the standard location for environments that allows easy activation. So we should drop the `-p conda`.

Reply:
Fixed in b855286.